modules/nw-controlling-dns-pod-placement.adoc

```diff
@@ -23,25 +28,49 @@ As a cluster administrator, you can use a custom node selector to configure the
 .Procedure
 
 * To allow the daemon set for CoreDNS to run on certain nodes, configure a taint and toleration:
+
+. Set a taint on the nodes that you want to control DNS pod placement by entering the following command:
++
+[source,terminal]
+----
+$ oc adm taint nodes <node_name> dns-only=abc:NoExecute
+----
 +
-. Modify the DNS Operator object named `default`:
+Replace `<node_name>` with the actual name of the node.
```

Comment:
```suggestion
<1> Replace `<node_name>` with the actual name of the node.
```

Reply:
I suggest adding callouts here, so its clearer that the text under the code block is important for the code above it